.github/workflows/release-binaries.yml

```diff
@@ -0,0 +1,172 @@
+name: Deploy binaries and create release
+on:
+  push:
+    tags:
+      - "v*"
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: 1
+
+jobs:
+  github_build:
+    name: Build release binaries
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - target: x86_64-unknown-linux-gnu
+            os: ubuntu-latest
+            name: prometheus_wireguard_exporter-x86_64-unknown-linux-gnu.tar.gz
```

Comment:
Could we have these without compression perhaps?

I just find it doesn't save that much space and I like downloading binaries directly